### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mgrachev**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -14,6 +14,9 @@ const APP_VERSION: &str = env!("CARGO_PKG_VERSION");
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    #[cfg(feature = "dotenv")]
+    dotenv::dotenv()?;
```
Comment (manifest) [inline src/main.rs:18]: Do we need a linter in the runtime?

File: src/app/notification_puller.rs [hunk 2]
```diff
@@ -177,10 +175,18 @@ async fn process_delayed_notifications(
...
-            // even if we erred we could've sent some notifications to nats before
-            // and should commit their deletions to db
-            break;
+        if let Err(e) = nats_client.publish(notification).await {
+            if let Ok(err) = e.downcast::<std::io::Error>() {
+                // Even if we erred we could have sent some notifications to nats before
+                // and should commit their deletions to db
+                // Exception: no responders - we should delete such notifications
+                // TODO: Maybe add a counter of attempts to the notification table
+                // and delete them if attempts > 10 for example?
```
Comment (manifest) [inline src/app/notification_puller.rs:184]: It doesn't seem the case for JetStream. Can you provide a reference with the description of the error?

Comment (manifest) [reply src/app/notification_puller.rs:184]: In our case, we assume the stream is always created beforehand.

File: src/main.rs (hunk 1 above)
Comment (mgrachev) [reply src/main.rs:18]: It's not a linter. This library loads environment variables from `.env` files in development environment.

File: src/app/notification_puller.rs (hunk 2 above)
Comment (mgrachev) [reply src/app/notification_puller.rs:184]: > It doesn't seem the case for JetStream. Can you provide a reference with the description of the error?

https://docs.nats.io/nats-concepts/core-nats/queue#no-responder

**Review by manifest — APPROVED** [review]

**Pull request merged**